0.download_data/1.process_JUMP_plates_with_CytoTable.py

```diff
@@ -0,0 +1,71 @@
+"""
+Processes JUMP plates from manifest using CytoTable
+to produce joined compartment data in Parquet format
+for use in downstream analysis.
+"""
+
+import pathlib
+import shutil
+
+import cytotable
+import pandas as pd
+from parsl.config import Config
+from parsl.executors import ThreadPoolExecutor
+from pyarrow import parquet
+
+# create plates directory if it doesn't exist already
+if not (plates_folder := pathlib.Path("./0.download_data/data/plates")).is_dir():
+    plates_folder.mkdir()
+
+# process each plate from the manifest individually
+for _, plate_name, plate_s3_path in pd.read_csv(
+    "./0.download_data/data/jump_dataset_location_manifest.csv", header=0
+).to_records():
+    print("Processing plate ", plate_name)
+
+    # create a folder for the plate
+    if not (
+        plate_folder := pathlib.Path(f"./0.download_data/data/plates/{plate_name}")
+    ).is_dir():
+        plate_folder.mkdir()
+
+    # if we don't have the cytotable output for a plate, process it
+    if not (
+        cytotable_output_path := pathlib.Path(
+            f"./0.download_data/data/plates/{plate_name}/{plate_name}.parquet"
+        )
+    ).is_file():
+        # process plate using CytoTable
+        cytotable_output_path = cytotable.convert(
+            source_path=plate_s3_path,
+            dest_path=cytotable_output_path,
+            dest_datatype="parquet",
+            source_datatype="sqlite",
+            chunk_size=8000,
```

Comment:
Is this a practical parameter?

Reply:
Thanks for checking on this - yes, I feel this is practical for the time being. I used similar within earlier work with the JUMP data which may be found here: https://github.com/WayScience/coSMicQC/blob/main/docs/src/examples/jump_umap_analysis_with_cosmicqc.py . I wonder if it would make sense to test this more generally to see how performance is impacted but hesitate to change something that worked in earlier iterations.